src/app/messages/en/attributionText.ts

```diff
@@ -62,15 +62,17 @@ export const defaultText = `
 </p>
 `;
 
-export const teaText = `
+export const specialText = `
 <p>
   Want to cite, share, or modify this book? This book is
   <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by/4.0/">
```

Comment:
why is there a link here? it might be wrong if the book isn't actually creative commons

Reply:
this link was hardcoded for tea books. I've removed it, but maybe it would be better to add more variables like `showLicenceLink` and `licenseLink`?